### PR TITLE
Refactors Ore Eating to Allow the Consumption of Material Sheets

### PR DESCRIPTION
**Pull request by KasparoVy**
Further builds on #7085, #7129 & #7160.

Refactors ore eating to facilitate the ethical consumption of refined minerals that are more conveniently obtainable on station.
Kindly suggested by hungry kobolds.

Also fixes the material name of Plastitanium glass throughout the codebase.

Still fully functional in the sense that you can use the eat ore verb to consume what's in your hand or just click yourself while holding a stack of refined minerals or a raw ore, or if someone else clicks you with 'em

When eating a stack of multiple sheets, you will only consume a single sheet at a time. No separation is necessary, just use the verb or click yourself with it in hand.

### CODE REVIEW

File: vorestation.dme [hunk 1]
```diff
@@ -1641,9 +1641,9 @@
 #include "code\modules\ai\ai_holder_targeting_vr.dm"
 #include "code\modules\ai\interfaces.dm"
 #include "code\modules\ai\say_list.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai_vr.dm"
-#include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai_vr.dm"
+#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai.dm"
```
Comment (Arokha) [inline vorestation.dme:1646]: Mlem

**Comment by Arokha** [comment]
> "Gem Eater", "Expensive Taste", "Ore Eater"

> Eats sheets

Though from a standpoint of 'I need to be able to eat something when I'm an ore eater otherwise it's annoying', it does make sense. Maybe the name could be changed to METALLOPHAGE (half kidding).

File: vorestation.dme (hunk 1 above)
Comment (KasparoVy) [reply vorestation.dme:1646]: Fug how'd this make it in

I'll clean up the diff

**Comment by KasparoVy** [comment]
I agree, ill adjust the proc names a bit.

**Comment by KasparoVy** [comment]
Holy sheet...

**Comment by KasparoVy** [comment]
Wicked, cheers!

**Pull request merged**